layouts/community/ergodox/zweihander-macos/keymap.c

```diff
@@ -8,6 +8,24 @@
 
 // Best viewed in Xcode in Menlo Regular or SF Mono.
 
+enum custom_keycodes {
+    NEWITEM = SAFE_RANGE,
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+    case NEWITEM:
+        if (record->event.pressed) {
+            // when keycode NEWITEM is pressed
+            SEND_STRING("\n- "); // starts a new item in a Markdown-style list
+        } else {
+            // when keycode NEWITEM is released
```

Comment:
You don't need this part, and can be removed. 
```suggestion
```

Reply:
I can understand why people would want to remove a do-nothing `else` clause, but I hate to remove it entirely without a trace. I've removed it in [7dd0c4f](https://github.com/qmk/qmk_firmware/pull/18737/commits/7dd0c4f7995bb724b6879d9f47e0a068728eddbc), but left in a comment explaining what an `else` clause would be useful for. What do you think of it now?